### ISSUE TITLE
Classic Fanatics generic art for active buttons

### ISSUE THREAD

**Issue by TheOrgg** [buttonmen-dev/buttonmen#2697]
Thanks to two nights of insomnia, I have the active-without-artwork Fanatics done up with the help of Fog's legacy assets.  Thanks to @blackshadowshade for allowing me the honor of using Fog's files.

Now these need to be looked over with a fine tooth'd comb to make sure the dice match up, the names don't bleed over the edge or into the frames/filligree, and any other things that stick out as plain wrong.

Already noticed a few minor errors, but fixed all but one.

As I said, these do need to be checked.  I'll start to work on the Inactive ones at some point.

![barswanian](https://user-images.githubusercontent.com/44233993/104982701-d49c1300-59d0-11eb-8e4f-5e89aca385d0.png)
![bobby5150](https://user-images.githubusercontent.com/44233993/104982703-d534a980-59d0-11eb-830e-af4eaaa6dd68.png)
![Coyote](https://user-images.githubusercontent.com/44233993/104982704-d534a980-59d0-11eb-9f0c-8a0b9e5ad5b3.png)
![Cristofore](https://user-images.githubusercontent.com/44233993/104982705-d5cd4000-59d0-11eb-9455-f5a6e3f8b052.png)
![Darthcliff](https://user-images.githubusercontent.com/44233993/104982706-d5cd4000-59d0-11eb-822e-86119cfda843.png)
![Discordia](https://user-images.githubusercontent.com/44233993/104982708-d5cd4000-59d0-11eb-95e2-87719002ea5e.png)
![fxdirect](https://user-images.githubusercontent.com/44233993/104982709-d5cd4000-59d0-11eb-843c-9fa967e424da.png)
![GorgorBey](https://user-images.githubusercontent.com/44233993/104982710-d665d680-59d0-11eb-9e51-86881d96fc71.png)
![grayhawk](https://user-images.githubusercontent.com/44233993/104982712-d665d680-59d0-11eb-8e2c-a682d187be52.png)
![GreatWolf](https://user-images.githubusercontent.com/44233993/104982714-d665d680-59d0-11eb-85ad-ed97bfdf7ae8.png)
![Grivan](https://user-images.githubusercontent.com/44233993/104982715-d6fe6d00-59d0-11eb-91f8-fc9f706e58ce.png)
![Hrodgar](https://user-images.githubusercontent.com/44233993/104982716-d6fe6d00-59d0-11eb-97fa-1b1fcfcf6751.png)
![IIconfused](https://user-images.githubusercontent.com/44233993/104982717-d6fe6d00-59d0-11eb-969c-ac0a4d324ffc.png)
![Jordan](https://user-images.githubusercontent.com/44233993/104982718-d7970380-59d0-11eb-8d3a-62d2b3743ed5.png)
![kestrel](https://user-images.githubusercontent.com/44233993/104982719-d7970380-59d0-11eb-9f6e-9fb832a9d52f.png)
![maga](https://user-images.githubusercontent.com/44233993/104982720-d7970380-59d0-11eb-91ef-38bd2675ad5b.png)
![nelde](https://user-images.githubusercontent.com/44233993/104982722-d7970380-59d0-11eb-848c-43bbc3802fbe.png)
![NoopMan](https://user-images.githubusercontent.com/44233993/104982723-d82f9a00-59d0-11eb-96c9-b0f81667ca86.png)
![Notorious](https://user-images.githubusercontent.com/44233993/104982724-d82f9a00-59d0-11eb-9688-06e01517a921.png)
![Pain](https://user-images.githubusercontent.com/44233993/104982725-d82f9a00-59d0-11eb-907d-49ff4060d2fa.png)
![Raupe](https://user-images.githubusercontent.com/44233993/104982726-d8c83080-59d0-11eb-8d50-495e37190ec2.png)
![SailorMur](https://user-images.githubusercontent.com/44233993/104982727-d8c83080-59d0-11eb-85c8-2758fbe66ed7.png)
![santiago](https://user-images.githubusercontent.com/44233993/104982728-d8c83080-59d0-11eb-99b5-ee25dfafb423.png)
![SC(TheDeuce)](https://user-images.githubusercontent.com/44233993/104982730-d960c700-59d0-11eb-8377-8cccb0a8bf0b.png)
![Snuff](https://user-images.githubusercontent.com/44233993/104982732-d960c700-59d0-11eb-9cbf-5a1f86fb4834.png)
![spindisc](https://user-images.githubusercontent.com/44233993/104982733-d960c700-59d0-11eb-96f9-3b3b826c3ed8.png)
![Squiddhartha](https://user-images.githubusercontent.com/44233993/104982734-d9f95d80-59d0-11eb-93a7-5bee53ae5b1e.png)
![Stick](https://user-images.githubusercontent.com/44233993/104982735-d9f95d80-59d0-11eb-811f-1b65022314d0.png)
![TheMachine](https://user-images.githubusercontent.com/44233993/104982737-d9f95d80-59d0-11eb-99a0-2b8c674047cb.png)
![Torch](https://user-images.githubusercontent.com/44233993/104982738-da91f400-59d0-11eb-9951-0155a8f3341c.png)
![Totoro](https://user-images.githubusercontent.com/44233993/104982739-da91f400-59d0-11eb-9507-6378bb757bbd.png)
![trifecta](https://user-images.githubusercontent.com/44233993/104982740-da91f400-59d0-11eb-8b06-2eea6c14e804.png)
![TYFTFB](https://user-images.githubusercontent.com/44233993/104982741-db2a8a80-59d0-11eb-97ff-827be1454ee1.png)
![wtrollkin](https://user-images.githubusercontent.com/44233993/104982742-db2a8a80-59d0-11eb-8a6d-a4ca79accd9a.png)
![zaph](https://user-images.githubusercontent.com/44233993/104982743-db2a8a80-59d0-11eb-8f72-246ca5590245.png)
![Zomulgustar](https://user-images.githubusercontent.com/44233993/104982744-db2a8a80-59d0-11eb-90c3-3a0893d4275a.png)

**Comment by TheOrgg** [comment]
Should Roupe have the big filligreee (like Zaph/Torch?)

**Comment by blackshadowshade** [comment]
Without having checked any of these, they look very pretty! Nice work!

**Comment by dwvanstone** [comment]
They look great!

What about icarus?

User kestrel still plays on our site - you might start a game with him and see if he has artwork. (He had artwork on the old site.)

Should IIconfused time & space die look more like a tetrahedron? (I noticed that was a theme for other time & space dice.)

> Should Roupe and Nelde have the big filligreee (like Zaph/Torch?)
> Should grayhawk have the smaller filligree?

Why do some get a big filigree and others the smaller? I assume that's up to you. Do what you like!

> Should I nudge Spindisc's name so it doesn't touch the edge?

Looks fine as is - do so if you feel like it, I'd say.

**Comment by TheOrgg** [comment]
The filligree had several examples.  I overlooked one at first.

And I didn't know how to indicate both a Time and Space and Warrior die for 2Confused (illConfused?  about that name I'm confused...).  Similarly, I didn't try making a Twin Time and Space or a Twin Warrior icon; PSP isn't easy to work with.

Icarus...  looks like I overlooked icarus.

**Comment by dwvanstone** [comment]
Then I think IIconfused is just fine! It's just icarus that would be a nice addition.

Thank you for your good work!!!

**Comment by TheOrgg** [comment]
I just totally overlooked Icarus.  At one point I copied over all the rest of the names to GIMP, and I think I just skipped it.

![icarus](https://user-images.githubusercontent.com/44233993/105103330-103ce880-5a76-11eb-8f68-034132606a47.png)
 
 
 Any thoughts on this for Plasma?  This was the most complex nonworking button on the list, so I tried doing it first.  The plasma ring is from something I found in Fog's incomplete folder and it seemed like it was his intent to use it as a die frame similar to Fire.
 
 
![150pxGryphon](https://user-images.githubusercontent.com/44233993/105106044-f18d2080-5a7a-11eb-9d2d-7e0ab5505dcf.png)

**Comment by TheOrgg** [comment]
![Gryphon](https://user-images.githubusercontent.com/44233993/105236614-bbf73e80-5b31-11eb-8b75-cbb07f3db99e.png)
Large Gryphon to show detail of the Plasma frame.  Suggestions very welcomed.

**Comment by blackshadowshade** [comment]
Didn't the last plasma die get changed to having a {p} instead of a {P}?

**Comment by TheOrgg** [comment]
Fog used a capital P for Poison and a capital N for Null in his design.  The p/P thing is hard to tell the difference with the font.  The n/N would have a bigger impact, but I kept the N for consistency.  I tried to make them as close to his designs as I could using the same assets he had in his files.

Right now I need feedback on the Plasma frame effect which he didn't have on any completed buttons.

**Comment by TheOrgg** [comment]
![150PlasmaduddExample](https://user-images.githubusercontent.com/44233993/105251078-755d1080-5b40-11eb-8186-865185318e56.png)
![PlasmaduddExample](https://user-images.githubusercontent.com/44233993/105251079-75f5a700-5b40-11eb-8514-c3fa6df645a0.png)

Let me rephrase.  Which of these six possible Plasma treatments looks best?

**Comment by jl8e** [comment]
The purple fills are hard to read because of the low contrast.

I think the fourth looks cleanest, followed by the first

**Comment by dwvanstone** [comment]
Fourth followed by first. Agreed.

**Comment by TheOrgg** [comment]
I was leaning towards the fourth as well after mocking them up.  Gimme a couple of days or so and I should have the rest o the Classics that need art.

Any other observations or opinions?

**Comment by jl8e** [comment]
I'd probably reduce the amount of blur inside the circle

**Comment by TheOrgg** [comment]
I have no clue how to reduce blur (nor fake 'shine' on the top left corner).  :(

Here's the rest of them.  Ya'll please check them for correctness and omissions/additions.  Prize goes for Tasselhoff B for most annoying button to create; had to redo three dice frames, figure out how a reserve warrior looks, and go back into Fog's files to make a significantly reduced-in-size name that'd fit on the button with all its dice frames allocated.

![antherem](https://user-images.githubusercontent.com/44233993/105449881-e08d0c80-5c3e-11eb-9c3c-83b8ff8050b3.png)
![anthony](https://user-images.githubusercontent.com/44233993/105449882-e125a300-5c3e-11eb-85d0-2abba3c478c1.png)
![Boot2daHead](https://user-images.githubusercontent.com/44233993/105449883-e125a300-5c3e-11eb-883d-51d2af1e6657.png)
![dwelsh](https://user-images.githubusercontent.com/44233993/105449884-e1be3980-5c3e-11eb-9e66-82183b415a66.png)
![echopapa](https://user-images.githubusercontent.com/44233993/105449887-e1be3980-5c3e-11eb-9b80-846885963c06.png)
![flesh99](https://user-images.githubusercontent.com/44233993/105449889-e1be3980-5c3e-11eb-876c-4824252acbd0.png)
![fnord](https://user-images.githubusercontent.com/44233993/105449891-e256d000-5c3e-11eb-9736-b530b42515ea.png)
![Gryphon](https://user-images.githubusercontent.com/44233993/105449892-e256d000-5c3e-11eb-9dc1-33320430d45e.png)
![inundator](https://user-images.githubusercontent.com/44233993/105449894-e256d000-5c3e-11eb-8e36-75cd3f2ac10b.png)
![Jennie](https://user-images.githubusercontent.com/44233993/105449895-e2ef6680-5c3e-11eb-89b8-1941665929b2.png)
![luke ii](https://user-images.githubusercontent.com/44233993/105449897-e2ef6680-5c3e-11eb-9090-21259be82d4a.png)
![moussambani](https://user-images.githubusercontent.com/44233993/105449899-e387fd00-5c3e-11eb-9433-9a74fb908ae7.png)
![NeoVid](https://user-images.githubusercontent.com/44233993/105449900-e387fd00-5c3e-11eb-89c7-03ce67e312f1.png)
![randomlife](https://user-images.githubusercontent.com/44233993/105449903-e387fd00-5c3e-11eb-821a-12e4cad0fd24.png)
![seeker](https://user-images.githubusercontent.com/44233993/105449904-e4209380-5c3e-11eb-9955-eee94500f8e1.png)
![SyberCat](https://user-images.githubusercontent.com/44233993/105449906-e4209380-5c3e-11eb-8aba-84efc1bbbf2a.png)
![TasselhoffB](https://user-images.githubusercontent.com/44233993/105449907-e4209380-5c3e-11eb-8f8b-16b581e2898e.png)
![Trouble](https://user-images.githubusercontent.com/44233993/105449910-e4b92a00-5c3e-11eb-9eb3-eec1fc124005.png)
![Tweedledum](https://user-images.githubusercontent.com/44233993/105449911-e4b92a00-5c3e-11eb-83af-802e222b2e58.png)
![vhoodoo](https://user-images.githubusercontent.com/44233993/105449912-e4b92a00-5c3e-11eb-9d35-507d48c76457.png)

**Comment by dwvanstone** [comment]
I didn't check as detailed as I probably should have, but I checked the # of dice, the basic sizes, the spelling of the names, and if you left any out. Looks good. Well done!

It makes me excited about the chaotic skill to bring in a bunch of these classic fanatics...

**Comment by TheOrgg** [comment]
Yup, @blackshadowshade  willing.  Add a few with the Wildcard and Plasma, and I think the whole set'll be active.

I finally put the Higher Quality images up on the sets wiki, and turns out I need to fix a few of the ones from Fog's files to remove the "flashing" from around the circle (and create about a dozen of the simpler ones that weren't in the files individually).

**Comment by TheOrgg** [comment]
![150lunatic](https://user-images.githubusercontent.com/44233993/105655060-f8ee6880-5e84-11eb-8885-253206ede64b.png)
Found the other minor error (Trip Option, not just Option).

**Comment by TheOrgg** [comment]
I went over the other Fanatics and found a few other minor things (dice frame inconsistency, bleeding over the outer circle), but here's a few that I felt were worthy of checking on:


ABCGi has an (X,Y)! on button, (X,X)! on recipe

Binder: 8/13 on image, 4/12 on recipe.

Cest Wha has a pk(2) on image, p(2) on recipe.

Gman has stinger but rage on image?

Prof. Booty has last two dice swapped, update recipe order?

RavenBlack-- Weak or Mighty 10?  Recipe/image mismatch.

Also, found two mistakes I made:
![150xdwelsh](https://user-images.githubusercontent.com/44233993/105735078-eada2f80-5ef8-11eb-8f14-4b90be6918ab.png)

![150inundator](https://user-images.githubusercontent.com/44233993/105735445-4f958a00-5ef9-11eb-992a-6184158f7b89.png)

**Comment by TheOrgg** [comment]
By Kestral's request, their fanatic with a picture of a Kestral instead of the classic fanatics background.  I used a picture of the postage stamp, as postage images are considered public domain.

![150pxKestralFanatic](https://user-images.githubusercontent.com/44233993/109178289-51e44180-774e-11eb-90db-5751fe68cc82.png)

**Comment by jl8e** [comment]
You probably want to tweak the twin die in some way; it looks fairly butt-like

**Comment by TheOrgg** [comment]
Eh, it's an Ornery Radioactive using Fog's dice frame designs and placement; I just put the kestrel in the place of the "Classic Fanatics" logo background.   The "butt" looks like  a crying face from The Binding of Isaac to me.

The lack of details in the dice frames (which I know you've noted before, and you're correct that it is annoying) is due to a combination of my inexperience with image editing and/or the limitations of a 150px image.

**Comment by TheOrgg** [comment]
So, got in touch with Stooooooooooooooooooooooooooooooo, and found out that his button is... caffeine.

He liked the idea of a Kestralization of his Fanatic, so here's his two choices.  I'll shrink down the one he likes best.

The one without the filigree feels empty in the corner, and even without filigree I can't really enlarge the molecule any further.  The background color is so the image 'pops' a bit more than it did on a white background.

![stooooo1](https://user-images.githubusercontent.com/44233993/110479615-390f5080-80ab-11eb-9057-5781dde83992.png)
![stooooo2](https://user-images.githubusercontent.com/44233993/110479619-39a7e700-80ab-11eb-8513-b25fe2b58446.png)

**Comment by TheOrgg** [comment]
![150pxStoooooFanatic](https://user-images.githubusercontent.com/44233993/110631771-81427780-816c-11eb-8df1-dc29d5a1c3f6.png)

**Comment by TheOrgg** [comment]
So while solving a problem with Squidhartha's original Fanatic artwork, I decided to take a mental break and go ahead and do Elihu Root's fanatic as well, even though he said "I suppose it would make sense to use that (an image of the historic figure Elihu Root) image, but it's not very important to me."  I had the programs open, so it didn't take that much extra time, and helped me figure out how to make SH's fanatic visible when shrunk to a smaller size.  (made it giant, blurred it a bit, stacked the layers to decrease the transparency of the edges of the image, then blur a bit more the bottom layer, stacking it only twice-- that looked so good I used it on the larger Fanatic, too).


Bigs, for the users in question to use as they want if they want.
![SquidharthaFanatic](https://user-images.githubusercontent.com/44233993/112164481-ea3ddc80-8bbb-11eb-9f0d-739d421284bd.png)

![ElihuRootFanatic](https://user-images.githubusercontent.com/44233993/112164594-02adf700-8bbc-11eb-830b-1fd7bc774521.png)

Smalls, or you'd kill me.
![150pxSquidharthaFanatic](https://user-images.githubusercontent.com/44233993/112165170-810a9900-8bbc-11eb-8972-4cc1656b83f9.png)

![150pxElihuRootFanatic](https://user-images.githubusercontent.com/44233993/112165426-bb743600-8bbc-11eb-869e-a98d17ff7309.png)

**Comment by Bassellope** [comment]
Here's a headshot for "Darrin":

![darrin_cowbell](https://user-images.githubusercontent.com/81323575/112359414-9d373480-8ca7-11eb-9af8-d6201d71ae06.jpg)

**Comment by TheOrgg** [comment]
That was surprisingly tricky, but I think I figured it out.  How's it look?

![DarrenFanatic](https://user-images.githubusercontent.com/44233993/112412018-cf6a8580-8ceb-11eb-92b3-5b3f3e1803bf.png)

![150pxDarrenFanatic](https://user-images.githubusercontent.com/44233993/112412072-ea3cfa00-8ceb-11eb-9b62-abb6a1d4fb2b.png)

**Comment by Bassellope** [comment]
Looks fantastic!  Thank you.

**Comment by blackshadowshade** [comment]
@TheOrgg, I'm happy to start adding these images, since it looks like you're going to continue adding images as you have time. Are they all ready to go? If so, I'll put up a pull request, and you can check that I have the correct final versions.

**Comment by blackshadowshade** [comment]
> I went over the other Fanatics and found a few other minor things (dice frame inconsistency, bleeding over the outer circle), but here's a few that I felt were worthy of checking on:
> 
> ABCGi has an (X,Y)! on button, (X,X)! on recipe
>
>
> Binder: 8/13 on image, 4/12 on recipe.
>
>
> 
> Cest Wha has a pk(2) on image, p(2) on recipe.
> 
> Gman has stinger but rage on image?
> 
> Prof. Booty has last two dice swapped, update recipe order?
> 
> RavenBlack-- Weak or Mighty 10? Recipe/image mismatch.
> 
> Also, found two mistakes I made:
> ![150xdwelsh](https://user-images.githubusercontent.com/44233993/105735078-eada2f80-5ef8-11eb-8f14-4b90be6918ab.png)
> 
> ![150inundator](https://user-images.githubusercontent.com/44233993/105735445-4f958a00-5ef9-11eb-992a-6184158f7b89.png)

I'll refer you to #2036, which already has some links and/or discussion on some of these mismatches.

**Comment by TheOrgg** [comment]
There's a little more discussion here on the mismatches:  https://github.com/buttonmen-dev/buttonmen/issues/2704  The only thing there that needs your attention as far as I can see is a new RavenBlack image that matches the recipe here on site.  I still suspect the recipe on site is an error, though.

Everything Fanatical should have at least a Generic, so get to it in your own time.  This isn't an issue that feels pressing, though I get that it's probably a quicker fix.

If I get in touch with other Fanatics who want images under the Fog dice frames and names, I'll put them here.  Considering there's only a couple more Fanatic people that've come and accepted the proposed games I've made, I don't expect more than one or two more "semi-generic" Fanatics to pop up.  If they do, I'll post them here.

Note that all the ones I'm replacing the Classic Fanatics logo with are under directive from the players themself; some, like KaijuGamer, didn't want his image messed with at this time.

**Comment by blackshadowshade** [comment]
Great, I'll get started at making a pull request for what you've got currently then.

**Comment by TheOrgg** [comment]
@Bassellope , looks like Fog originally had your recipe messed up and BSS just noticed it.  Just in case you're wanting to get copies made, here's the large corrected file.
![DarrenFanatic](https://user-images.githubusercontent.com/44233993/112719727-55dbbe80-8ec8-11eb-8180-88684e9b0546.png)

**Comment by blackshadowshade** [comment]
Once you get me a small version of that image, @TheOrgg, I'll update it on the pull request.

**Comment by TheOrgg** [comment]
Updated on the pull request.  Also, I cracked Fog's shine technique while looking at the pull request.  Duplicate the shadow, put it at the front, and adjust it until the "shine" effect comes out.  15% opacity is what worked when I tried it out.  You need to lighten the artwork up some for this to work well, though.  Here's an example using Darrin.  
![11DarrinExampleShine](https://user-images.githubusercontent.com/44233993/112723127-0e126280-8edb-11eb-92be-2ae607b3afed.png)

**Comment by TheOrgg** [comment]
I checked the Classic Fanatics page updated today, and there's several new arts that were not put in over the older artworks.

Spotted specifically were ElihuRoot, Darrin, and JeffBoyArDee who had updates to add artwork or correct an incorrect recipe.  Several others seem to still have the older pictures as well, such as Hauruspex and several others.  I exported all of them with the new shine/shade to have the full set look consistent.

Any reason for the omissions?

**Comment by TheOrgg** [comment]
I did ctrl f5 like Google said to do.  Check Calmon on your main site.  The updated one has yellow dice frames.

**Comment by dwvanstone** [comment]
Calmon on the main site has yellow dice frames. I think it's some kind of browser caching issue you're seeing. Maybe try on another browser? 

![Screen Shot 2021-04-19 at 4 33 16 PM](https://user-images.githubusercontent.com/6751265/115299631-071eee80-a12d-11eb-9b14-2f75f7850a99.png)

**Comment by blackshadowshade** [comment]
Yup, I'd think this is a caching issue. Wait a day or two then check again. Or try Shift-Reload to do a forced reload.

**Comment by TheOrgg** [comment]
The shift-reload/ctrl f5 didn't work, but deleting the cached files did work.  Sorry for bothering ya'll about it.  :)
